src/tests/Microsoft.PowerFx.Core.Tests/TestRunnerTests/InternalSetupTests.cs

```diff
@@ -100,13 +100,13 @@ public void InternalSetup_Parse_HandlerAndTwoFlags2()
         }
 
         [Fact]
-        public void InternalSetup_Parse_TableSyntaxDoesntWrapRecordsFlags()
+        public void InternalSetup_Parse_DisableTableSyntaxDoesntWrapRecordsFlags()
```

Comment:
Could you also add a test for both flags disabled in case we need to disable multiple default features in the future?

Reply:
Added that and a few more tests